src/Core/tests/DeviceTests/Stubs/LayoutStub2.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Maui.DeviceTests.Stubs
+{
+	public class LayoutStub2 : LayoutStub
+	{
+		public override FlowDirection FlowDirection { get; set; } = FlowDirection.LeftToRight;
```

Comment:
For the new test added, i need a `LayoutStub`  that returns the real value set for `FlowDirection`